rollup.config.ts

```diff
@@ -0,0 +1,156 @@
+import { babel } from '@rollup/plugin-babel';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import path from 'path';
+import { RollupOptions } from 'rollup';
+import dts from 'rollup-plugin-dts';
```

Comment:
I would've used `rollup-plugin-dts` for our config before, but it broke on some types. I love what it does and it might work now but I'm reluctant to use it in tRPC because of how crazy the types can get.

Reply:
Oh didnt know. Will look into that and make sure its correct 👍